samples/wifi/shell/README.rst

```diff
@@ -78,6 +78,22 @@ The following is an example of the CLI command:
 
        west build -b nrf9160dk/nrf9160/ns -- -DEXTRA_CONF_FILE=overlay-scan-only.conf -DSHIELD=nrf7002ek
 
+To build for the Thingy:91 X to be run on the nRF5340, use the ``thingy91x/nrf5340/cpuapp`` board target with the ``SB_CONFIG_THINGY91X_STATIC_PARTITIONS_NRF53_EXTERNAL_FLASH=y`` CMake option set.
```

Comment:
```suggestion
To build for the Thingy:91 X to be run on the nRF5340 DK, use the ``thingy91x/nrf5340/cpuapp`` board target with the ``SB_CONFIG_THINGY91X_STATIC_PARTITIONS_NRF53_EXTERNAL_FLASH=y`` CMake option set.
```

Reply:
This is not about running on the DK, the board is the Thingy:91 X. Should I restructure the sentence there?

Reply:
Yes, so that it becomes a bit clearer.